Ok, this does not make sense for me, it actually works for @sanjaysrikakulam on the test server but fails here with the same error that we have seen previously.

```
- Generated html report: file:///home/runner/work/galaxy/galaxy/galaxy%20root/run_unit_tests.html -
=========================== short test summary info ============================
FAILED test/unit/app/tools/test_metadata.py::TestMetadata::test_simple_output_directory - AttributeError: object_store_cache_size
FAILED test/unit/app/tools/test_metadata.py::TestMetadata::test_simple_output_extended - AttributeError: object_store_cache_size
FAILED test/unit/app/tools/test_metadata.py::TestMetadata::test_primary_dataset_output_extension_directory - AttributeError: object_store_cache_size
FAILED test/unit/app/tools/test_metadata.py::TestMetadata::test_primary_dataset_output_metadata_override_directory - AttributeError: object_store_cache_size
FAILED test/unit/app/tools/test_metadata.py::TestMetadata::test_primary_dataset_output_metadata_override_extended - AttributeError: object_store_cache_size
FAILED test/unit/app/tools/test_metadata.py::TestMetadata::test_list_discovery_extended - AttributeError: object_store_cache_size
```

--- a/lib/galaxy/model/unittest_utils/data_app.py
+++ b/lib/galaxy/model/unittest_utils/data_app.py
@@ -61,6 +61,7 @@ class GalaxyDataTestConfig(Bunch):
         self.object_store = "disk"
         self.object_store_check_old_style = False
         self.object_store_cache_path = "/tmp/cache"
+        self.object_store_cache_size = -1
         self.object_store_store_by = "uuid"
 
         self.umask = os.umask(0o77)

--- a/lib/galaxy/objectstore/__init__.py
+++ b/lib/galaxy/objectstore/__init__.py
@@ -1726,6 +1726,7 @@ def config_to_dict(config):
         "jobs_directory": config.jobs_directory,
         "new_file_path": config.new_file_path,
         "object_store_cache_path": config.object_store_cache_path,
+        "object_store_cache_size": config.object_store_cache_size,
         "gid": config.gid,
     }
 

